lib/config_servers.js

```diff
@@ -126,9 +126,9 @@ class ConfigServers extends EventEmitter {
         });
 
         // Boot all the servers
-        for (var i = 0; i < self.managers.length; i++) {
-          yield self.managers[i].start();
-        }
+        yield self.managers.map(function(server) {
```

Comment:
Does this do what you think it does? I think you mean to have something like:
```
yield Promise.all(self.managers.map(server => server.start()));
```
Otherwise you are `yield`ing the creation of an array of promises, which will be immediate.

Reply:
Yes, it looks like you're doing this in a few places here. Since we already depend on `bluebird` you can also use:
```
yield Promise.all(self.managers, server => server.start());
```

Reply:
I **believe** it does, but I'm also a novice with this...

According to co, it does: https://github.com/tj/co#arrays

But either way **should** work.

Reply:
Ah, that's.. magical :) I would prefer to be explicit about it, I don't think `await` will do that with an array of promises.  Sorry for the typo there